### PR TITLE
VACMS-6341: Top task content types.

### PR DESCRIPTION
**Pull request by kevwalsh**
## Description

Relates to #6341.

## Testing done

Behat, visual.

## QA steps

For now, none of the editorial experience here should be visible to non-content-admin or admin users

## test content model

- [x] Set three Services taxonomy terms to "non-clinical services"

1. Go to /taxonomy/term/1026/edit, /taxonomy/term/1025/edit, and /taxonomy/term/1027/edit
2. You should be able to set these each to VAMC type of care: non-clinical service.

- [x] Make sure these are not exposed in current UI

3. Go to /node/add/regional_health_care_service_des ... make sure that terms 1025, 1026, and 1027 are not visible from the dropdown (they have been excluded via the change to the entity reference source view. 

- [ ] Test non-clincal health services

4. Go to /node/add/vha_facility_nonclinical_service, you should see these three taxonomy terms from the Service dropdown
5. You should be able to pick a facility, and a section. 
6. Go ahead and add three facility non-clinical service nodes, one for each of the 3 services. Assign to Aberdeen VA clinci and to Section: VA Sioux Falls (in VISN 23). 

- [ ] See that top task pages are now "service aware"

7. Create one of each of three top task pages (VAMC system Medical records, VAMC system Billing and insurance, and VAMC system Register for care), and again assign field_office and Section to VA Sioux Falls. 
8. Note that  `field_service_name_and_descri` has been set for each of these via their default field settings. if you want, you can unhide the field on the content type form, go back and edit the node, and see that it has been populated as expected with the relevant Service.  

## Test GraphQl feasibility

This should be enough data to generate a GraphQL query to use a node from any of the three content type content types to get: 

1. a VAMC system node ID via `field_office` (in this case, 8135 for VA Sioux Falls)
2. a single taxonomy term for `field_service_name_and_descri` (1025, 1026, or 1027)
3. an array of VAMC facility non-clinical service nodes associated to facilities that also have `field_region_page` set to field_office` and `field_service_name_and_descri` set to the taxonomy term. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ethanteague** [comment]
@kevwalsh QA is failing on `node/add/regional_health_care_service_des` when I go to `/node/add/vha_facility_nonclinical_service` the Service dropdown is empty.
image

**Comment by kevwalsh** [comment]
> @kevwalsh I fixed the tests, but QA is failing on `node/add/regional_health_care_service_des` when I go to `/node/add/vha_facility_nonclinical_service` the Service dropdown is empty. 

you have to edit the taxonomy terms first and add the Type of Care. Did you do that after updating behat tests?

**Comment by ethanteague** [comment]
> > @kevwalsh I fixed the tests, but QA is failing on `node/add/regional_health_care_service_des` when I go to `/node/add/vha_facility_nonclinical_service` the Service dropdown is empty.
> 
> you have to edit the taxonomy terms first and add the Type of Care. Did you do that after updating behat tests?

Yep, did that. I did the qa before pushing up the test fixes.

File: config/sync/field.field.node.vamc_system_medical_records_offi.field_fax_number.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - field.storage.node.field_fax_number
+    - node.type.vamc_system_medical_records_offi
+  module:
+    - telephone
+id: node.vamc_system_medical_records_offi.field_fax_number
+field_name: field_fax_number
+entity_type: node
+bundle: vamc_system_medical_records_offi
+label: 'Fax number'
+description: 'The number to which Veterans can fax signed VA Form 10-5345a.'
```
Comment (swirtSJW) [inline config/sync/field.field.node.vamc_system_medical_records_offi.field_fax_number.yml:15]: Minor: Seems odd to entangle a specific form ID in the help text for a fax number.

**Review by swirtSJW — APPROVED** [review]

File: config/sync/field.field.node.vamc_system_medical_records_offi.field_fax_number.yml (hunk 1 above)
Comment (kevwalsh) [reply config/sync/field.field.node.vamc_system_medical_records_offi.field_fax_number.yml:15]: That help text can be updated later. But hopefully this field will be depracated altogether soon, VA wants to phase out Fax services.

**Pull request merged**